suricata/update/rule.py

```diff
@@ -310,7 +313,12 @@ def parse_fileobj(fileobj, group=None):
         try:
             rule = parse(buf, group)
             if rule:
-                rules.append(rule)
+                try:
+                    if rule["sid"]==None:
```

Comment:
PEP8 standards still not followed. Spaces around operators. Again, was this `==None` necessary?